src/config.ts

```diff
@@ -0,0 +1,14 @@
+export interface IConfig {
+  serviceUrl: string;
+  rustc: string;
+}
+
+const configUrl = "./config.json";
+
+export default async function getConfig() {
+  if (!this.config) {
+    this.config = await fetch(configUrl).then(resp => resp.json());
+  }
+
+  return this.config;
```

Comment:
what is `this` in this context? for a module, you can declare the variable above